### DISCUSSION TITLE
Run dovi_tool on raspberry pi

### DISCUSSION THREAD

**Discussion by MelvinSDRS** [quietvoid/dovi_tool#261] [Q&A] [answered]
First thanks a lot for your work. I want to run dovi_tool on raspberry pi but the Linux asset that I can find in the releases is not for arm processors so the binary can not be launched on rpi. Do you have a workaround on how I can use the tool on linux arm? Thank you!

**Comment by quietvoid** [comment, marked as answer]
You can always clone the source and build it manually.

**Reply by MelvinSDRS** [reply]
Okay thank you!

**Comment by quietvoid** [comment]
There are now aarch64/ARM64 Linux binaries.
